ci/integration_tests.sh

```diff
@@ -10,8 +10,9 @@ export GRADLE_OPTS="-Xmx2g -Dorg.gradle.jvmargs=-Xmx2g -Dorg.gradle.daemon=false
 export SPEC_OPTS="--order rand --format documentation"
 export CI=true
 
-# Source shared function for splitting integration tests
-source "$(dirname "${BASH_SOURCE[0]}")/get-test-half.sh"
+# Option for running in fedramp high mode
+FEDRAMP_FLAG=""
+[ "$FEDRAMP_HIGH_MODE" == "true" ] && FEDRAMP_FLAG="-PfedrampHighMode=true"
```

Comment:
Could we continue the pattern of using `ORG_GRADLE_PROJECT_fedrampHighMode`? This would allow us to avoid explicitly adding it to each invocation of gradle.

```suggestion
export ORG_GRADLE_PROJECT_fedrampHighMode="${FEDRAMP_HIGH_MODE:=false}"
```

Reply:
I thought of that, but i was thinking that keeping that as an "internal" implementation detail might be more clear. I was kind of envisioning that we would stick with always using the explicity property in the invocation and only rely on the env var when sub processes use that. 

I could for sure be convinced otherwise though.